forge-gui/res/cardsfolder/a/angel_of_condemnation.txt

```diff
@@ -5,7 +5,7 @@ PT:3/3
 K:Flying
 K:Vigilance
 A:AB$ ChangeZone | Cost$ 2 W T | ValidTgts$ Creature.Other | Mandatory$ True | TgtPrompt$ Select another target creature | Origin$ Battlefield | Destination$ Exile | RememberChanged$ True | SubAbility$ DelTrig | SpellDescription$ Exile another target creature. Return that card to the battlefield under its owner's control at the beginning of the next end step.
-SVar:DelTrig:DB$ DelayedTrigger | Mode$ Phase | Phase$ End of Turn | Execute$ TrigBounce | RememberObjects$ RememberedLKI | TriggerDescription$ Return exiled creature to the battlefield. | SubAbility$ DBCleanup
+SVar:DelTrig:DB$ DelayedTrigger | Mode$ Phase | Phase$ End of Turn | Execute$ TrigBounce | RememberObjects$ RememberedLKI | TriggerDescription$ Return the card to the battlefield. | SubAbility$ DBCleanup
```

Comment:
might be nice with `TriggerDescription$ Return {c:DelayTriggerRememberedLKI} to the battlefield under its owner's control at the beginning of the next end step.` but I don't know if that token would work yet